index.html

```diff
@@ -0,0 +1,127 @@
+            </div>
+        </section>
+        <section class="program-section">
+            <h3>Main Program</h3>
+            <div class="prog-main-cont">
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/lecture.png" alt="">
+                    <h4 class="prog-title">Lecture</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/exhibition.png" alt="">
+                    <h4 class="prog-title">CC Exhibition</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/camunication.png" alt="">
+                    <h4 class="prog-title">Forum</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/workshop.png" alt="">
+                    <h4 class="prog-title">Workshop</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+                <div class="prog-card">
+                    <img class="prog-img" src="./img/ignite.png" alt="">
+                    <h4 class="prog-title">CC Ignite</h4>
+                    <p class="prog-content">
+                        It is a long established fact that a reader will be distracted 
+                        by the readable content of a page when looking at its layout.
+                    </p>
+                </div>
+            </div>
+            <a class="prog-hole" href="#">See the Hole Program</a>
+            <button class="prog-btn">Joid CC Global Summit 2015</button>
+        </section>
+        <section class="speaker-section">
+            <h3 class="speaker-title">Featured Speakers</h3>
+            <div class="speaker-main-cont">
+                <div class="speaker-cards">
+                </div>
+                <button class="speaker-more-btn">More<img class="down-arrow" src="img/dwon-arrow.png"></img></button>
```

Comment:
- [ ] [OPTIONAL] it would be better if you changed `More` to `Less` after the user clicked on the button so the button text is meaningful

![Screenshot_20221203_093028](https://user-images.githubusercontent.com/25227022/205432499-b3f3298d-cf21-4700-9df6-f87fa88734dd.png)